src/main/java/net/countercraft/movecraft/combat/features/tracking/events/CraftSunkByEvent.java

```diff
@@ -56,7 +61,17 @@ public String causesToString() {
         StringBuilder stringBuilder = new StringBuilder();
         stringBuilder.append(((PlayerCraft) this.craft).getPilot().getDisplayName());
         stringBuilder.append(" ").append(I18nSupport.getInternationalisedString("Killfeed - Sunk By")).append(" ");
-        stringBuilder.append(NameUtils.offlineToName(latestDamage.getCause()));
+        Player cause = (Player) latestDamage.getCause();
```

Comment:
This could be a problem if the cause logs off, hence why it returns `OfflinePlayer` instead of `Player`.